satellite/accounting/live/live_test.go

```diff
@@ -5,6 +5,7 @@ package live
 
```

Comment:
nit: tests usually should be using `_test` suffix to ensure it's not relying on non-exposed behavior